index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
- [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.

![image](https://user-images.githubusercontent.com/62290545/221864160-f214920d-719e-4720-9396-5ea36cac8256.png)
![image](https://user-images.githubusercontent.com/62290545/221864332-34b241b2-431e-421d-988e-81054c8484ab.png)
![image](https://user-images.githubusercontent.com/62290545/221864507-34c177f6-49d4-4fbc-a672-72f713f8ccb4.png)